README.md

```diff
@@ -37,6 +39,7 @@ Options:
       --tag-pattern [regex]           # override regex pattern for release tags
       --tag-prefix [prefix]           # prefix used in version tags, default: v
       --starting-commit [hash]        # starting commit to use for changelog generation
+      --sort-commits-by-date          # sort commits by date (descending) instead of sorting by relevance
```

Comment:
Maybe this would be better as a generic option that took a value of “date“, for extensibility?

Reply:
Thanks for the quick review! Yeah, that makes sense. I have no time to do it now though. Feel free to suggest an edit or commit yourself or whatever :)

Reply:
Yeah a generic `--sort-commits` sounds good. I'll give it a shot.